### PR TITLE
Proofreading of Introduction.

### PR DESCRIPTION
**Pull request by TheMSB**
Several fixes to the existing Chapters.

### CODE REVIEW

**Comment by wandernauta** [comment]
:+1: from me.

**Comment by wandernauta** [comment]
Other than the two comments above, this lgtm.

**Comment by TheMSB** [comment]
> Not a typo: I was volunteered, if I remember correctly.

If you say so, I remembered you volunteering yourself but I reverted the change.

File: Report/Chapters/Design.tex [hunk 1]
```diff
@@ -52,7 +52,7 @@ \subsubsection{Menus}
 
 The primary menu (or `function menu') uses `drawers' for the categories of the functions. \index{primary menu} \index{function menu}
 This way it is easy to find functions even without an extensive knowledge of Haskell.
-In- and ontput blocks and the function definition block can be created using buttons at the bottom of the menu.
+In- and output blocks and the function definition block can be created using buttons at the bottom of the menu.
 This location is chosen to make them quickly available.
 The primary menu can be opened multiple times at once and moved around the screen so multiple users can work on a single program.
```
Comment (i3anaan) [inline Report/Chapters/Design.tex]: Ans also in order to never be limitted by screen size (as opposed to when it is stuck on an edge)

File: Report/Chapters/Design.tex [hunk 2]
```diff
@@ -73,7 +73,7 @@ \subsection{Behaviour and interaction design}
 
 \section{Languages and libraries}
 
-Viskell is written in Java using Java 8 with JavaFX. \index{JavaFX}
+Viskell is written in Java 8 that comes bundled with JavaFX. \index{JavaFX}
 Java 8 is chosen because it, unlike Java 7, still receives public updates at the moment of writing.
 Language level 8 is chosen to make use of some of the new features in this version.
```
Comment (i3anaan) [inline Report/Chapters/Design.tex]: Those 2 sentences are very alike

File: Report/Chapters/Implementation.tex [hunk 3]
```diff
@@ -41,7 +41,7 @@ \subsubsection{Expressions}
 This class is very simple and does not include more than the name of the function (so it can be called) and a way to lookup the type.
 
 Values are represented in a similar way, a \code{Value} object keeps track of the string representation of the value and its type. \index{Value}
-The string representation of the value should be written in a way that GHCi natively understands what is meant by it, altough this is not validated.
+The string representation of the value should be written in a way that GHCi natively understands what is meant by it, although this is not validated.
```
Comment (i3anaan) [inline Report/Chapters/Implementation.tex]: Isn't this validated by now?

File: Report/Chapters/Introduction.tex [hunk 4]
```diff
@@ -2,6 +2,6 @@ \chapter{Introduction}
...
 
-In this report the different aspects of the developing process are being discussed. The upcoming chapters discuss the requirements analysis, the design process, the implementation process (front end \& back end), how the application has been tested, the evaluation of the process and the conclusions.
+This report discusses different aspects of the development process. The upcoming chapters discuss the requirements analysis, design process, implementation process (front end \& back end), how the application has been tested, the evaluation of the process and the conclusions.
```
Comment (i3anaan) [inline Report/Chapters/Introduction.tex]: I like the introduction

File: Report/Chapters/Design.tex (hunk 2 above)
Comment (TheMSB) [inline Report/Chapters/Design.tex]: > hose 2 sentences are very alike

They are but one marginally improves upon the other.

File: Report/Chapters/Implementation.tex (hunk 3 above)
Comment (TheMSB) [inline Report/Chapters/Implementation.tex]: > Isn't this validated by now?

Don't know but that's not the point of this PR.

File: Report/Chapters/Design.tex (hunk 2 above)
Comment (i3anaan) [inline Report/Chapters/Design.tex]: I dont know, I find the last sentence to be very unecessary, id say either combine the 2 or drop the second one

**Pull request merged**